pkgs/applications/audio/jamulus/default.nix

```diff
@@ -0,0 +1,36 @@
+{ mkDerivation, stdenv, fetchFromGitHub, pkgconfig, qtscript, qmake, libjack2
+}:
+
+mkDerivation {
+  pname = "jamulus";
+  version = "3.5.3";
+  src = fetchFromGitHub {
+    owner = "corrados";
+    repo = "jamulus";
+    rev = "5385f4f113884537c18109e01791149754fe96d5";
+    sha256 = "0wrg0g2cligk3pd82h2sh38xgfbdkwklinrk4p0f4iyfiqfzf0ck";
+  };
+
+  nativeBuildInputs = [ pkgconfig qmake ];
+  buildInputs = [ qtscript libjack2 ];
+
+  buildPhase = ''
+    make clean
+    make
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp Jamulus $out/bin/Jamulus
+  '';
+
+  enableParallelBuilding = true;
+
+  meta = {
+    description = "Enables musicians to perform real-time jam sessions over the internet. You also need to enable JACK and should enable several real-time optimizations. See project website for details";
```

Comment:
Maybe some of this would better fit into `longDescription`.

Reply:
moved sentence 2 and 3 to longDescription